pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -32,7 +32,7 @@
 from pulser.register.base_register import BaseRegister, QubitId
 from pulser.register.mappable_reg import MappableRegister
 from pulser.register.register_layout import RegisterLayout
-from pulser.register.traps import COORD_PRECISION
+from pulser.register.traps import COORD_DECIMAL_PRECISION
```

Comment:
I'm fine with the renaming but I'm not 100% this constant is not used outside of `pulser`. 
To be safe, we would need to keep an alias `COORD_PRECSION = COORD_DECIMAL_PRECSION`, which kinda defeats the purpose